hwsc_file_transaction_svc.py

```diff
@@ -10,50 +10,41 @@
 CHUNK_SIZE = 1024 * 1024
 
 def download_chunk(reqFile):
-    with open(reqFile, 'readFile')as f:
+    with open(reqFile, 'rb')as f:
         while True:
             chunk = f.read(CHUNK_SIZE);
             if len(chunk) == 0:
                 return
             yield hwsc_file_transaction_svc_pb2.chunk(buffer=chunk)
 
-def upload_chunk(chunk, reqFile):
-    with open(reqFile,'writeFile') as f:
-        for line in chunk:
-            f.write(line.buffer)
-
-class FileTransactionClient:
-    def __init__(self,address):
-        channel = grpc.insecure_channel(address)
-        self.stub = hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceStub(channel)
-
-    def upload(self,target_file):
-        chunk = download_chunk(target_file)
-        response = self.stub.UploadFile(chunk)
-        assert response.length == os.path.getsize(target_file)
-
-    def download(self,target_file,response_file):
-        response = self.stub.DownloadFile(hwsc_file_transaction_svc_pb2.Request(name=target_file))
-        upload_chunk(response,response_file)
+def save_chunks_to_file(chunks, filename):
+    with open(filename,'wb') as f:
+        for chunk in chunks:
+            f.write(chunk.buffer)
 
 class FileTransactionService(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
     def __init__(self):
 
         class Servicer(hwsc_file_transaction_svc_pb2_grpc.FileTransactionServiceServicer):
             def __init__(self):
-                # pass
-                self.tmp_file_name = ''
+                pass
 
             def GetStatus(self, request, context):
                 print("Get Status")
 
-            def download(self,downloadRequest,context):
+            def DownloadZipFiles(self,downloadRequest,context):
                 if downloadRequest.name:
                     return download_chunk(self.tmp_file_name)
 
-            def upload(self,uploadRequest,context):
-                upload_chunk(uploadRequest,self.tmp_file_name)
-                return hwsc_file_transaction_svc_pb2.Reply(length=os.path.getsize(self.tmp_file_name))
+            def UploadFile(self, request_iterator, context):
+                print("[INFO] Requesting UploadFile service")
+                #save_chunks_to_file(request_iterator,'dummy_img.jpg')
+                save_chunks_to_file(request_iterator, context.fileName)
+                #return hwsc_file_transaction_svc_pb2.FileTransactionResponse(message='Hello', url='world', length=64)
+                return hwsc_file_transaction_svc_pb2.FileTransactionResponse(
```

Comment:
this is not the proper value for the field message. please refer to https://github.com/faraonc/hwsc-document-svc/blob/master/service/service.go

additionally, please use the status code.